lib/rubocop/cop/rspec/leaky_constant_declaration.rb

```diff
@@ -121,6 +124,16 @@ def on_module(node)
         def inside_describe_block?(node)
           node.each_ancestor(:block).any? { |ancestor| spec_group?(ancestor) }
         end
+
+        def defined_on_explicit_namespace?(node)
+          if node.is_a?(RuboCop::AST::ClassNode) ||
+              node.is_a?(RuboCop::AST::ModuleNode)
+
+            node.loc.name.source.include?('::')
```

Comment:
What would be node.namespace for cases without a namespace, cbase (::Const) and some explicit parent module?
I’d rather prefer  a check against namespace rather than the text source.